oe_translation.install

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa Translation install file.
+ */
+
+use Drupal\Core\Entity\Entity\EntityFormDisplay;
+
+/**
+ * Implements hook_install().
+ */
+function oe_translation_install() {
+  // Hide the "translation skills" fields on the User forms as we are not using
+  // the local translator plugin in its original form.
+  /** @var \Drupal\Core\Entity\Display\EntityFormDisplayInterface[] $form_displays */
+  $form_displays = EntityFormDisplay::loadMultiple(['user.user.default', 'user.user.register']);
+  foreach ($form_displays as $form_display) {
+    $component = $form_display->getComponent('tmgmt_translation_skills');
+    if ($component) {
+      $form_display->removeComponent('tmgmt_translation_skills');
+      $form_display->save();
+    }
+  }
+}
```

Comment:
Exactly and it's on purpose. We are removing those fields because we don't ever want to use the original local translator. We don't want fields on the user form used for specifying the language skills.

Reply:
Like this when uninstalling `oe_translation`, form fields will still be missing.
With a form alter they will be back again.

Reply:
@dxvargas It is configuration. It can be easily added back. When uninstalling, we don't know where to put those fields back. Also, it is our module that brings tmgmt in so it stands to reason that without our module, no tmgmt either,